### PR TITLE
Mark D110 as fully supported

### LINKED ISSUE #4: Testing with D110
Hello,

I tried your module on a D110 with a bluetooth connection using the following command : 
`python -m niimprint -c bluetooth -a "02:07:F4:4E:09:A6" -d 1 -r 90 -i examples/B21_30x15mm_240x120px.png`

Here is the result : 

Looks promising ! Thanks a lot for your work

### PR DESCRIPTION
**Pull request by Orhideous**
Also added layouts for various tapes.

Closes #4

### CODE REVIEW

File: examples/D110/94x168.png
Comment (AndBondStyle) [inline examples/D110/94x168.png:1]: I think this is too much images... Maybe we can make a simple image generator instead, or `--test-image` flag in the CLI that prints test image with specified resolution

File: readme.md [hunk 1]
```diff
@@ -57,26 +57,51 @@ It seems like B21 (and maybe other models?) has two bluetooth adresses. For me,
...
 
+### D110
+
+Pair device that starts from `02:13:` (identifies itself as printer).
+
+`python niimprint -m d110 -c bluetooth -a "02:13:FF:FF:FF:FF" -d 3 -r 90 -i examples/D110/96x172.png`
+
+| Tape type   | Printable dimensions (mm) | Pixels | Layout file       | Notes        |
+|-------------|---------------------------|--------|-------------------|--------------|
+| T14*22      | 13.5×21.5                 | 96×172 | `D110/96x172.png` |              |
```
Comment (AndBondStyle) [inline readme.md:88]: Should probably make a table of supported printers also. Table of labels is a good idea, maybe we can extend it with links to official label catalog (if there is a one?) and RFID tag values (some initial info here: https://github.com/kjy00302/niimprint/issues/4)

File: niimprint/models.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+SUPPORTED_DEVICES = {
+    "D11": Device("D11", 3, 96),
+    "D110": Device("D110", 3, 96),
+    "B1": Device("B1", 5, 384),
+    "B21": Device("B21", 5, 384),
+}
+
+
+class RejectionError(BaseException):
```
Comment (AndBondStyle) [inline niimprint/models.py:19]: Assert / RuntimeError is probably fine for now... or add a generic `NiimPrintError`

File: examples/D110/94x168.png
Comment (Orhideous) [reply examples/D110/94x168.png:1]: PR for this is underway.

**Pull request closed without merging**